web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import { Typography } from '@material-ui/core';
+import "./ViewFeedbackResponses.css";
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import AvatarComponent from '../avatar/Avatar'
+import MoodBadIcon from '@material-ui/icons/MoodBad';
+import InsertEmoticonIcon from '@material-ui/icons/InsertEmoticon';
+import SentimentSatisfiedIcon from '@material-ui/icons/SentimentSatisfied';
+//note that request id will be in actual object, so you will need to get information out of request id, template id and state
+
+
+const useStylesCardContent = makeStyles({
+    root: {
+      '&:last-child': {
+        paddingBottom: '16px',
+      }
+    }
+  }, { name: "MuiCardContent" })
+
+
+
+
+const ViewFeedbackResponses = (props) => {
+     useStylesCardContent();
+     let questions =[
+        {
+            id: 1,
+            questionContent: "What is your current knowledge about opossums?",
+            orderNum: 1,
+    
+        },
+        {
+            id: 2,
+            questionContent: "Do you think opossums are misunderstood creatures? Why?",
+            orderNum: 2,
+    
+        },
+        {
+          id: 3,
+          questionContent: "If you knew that opossums didn't carry rabies or other common 'vermin' diseases, are often very docile, and can eat up to 5,000 ticks a season, would your opinion change about opossums?",
+          orderNum: 3,
+    
+      },
+    
+    
+    ]
+    //note that submitter name will not be in actual returned object, but this is intermediary for time  being without api
+    let responses= [
+        {
+            answer: "I don't know that much about opossums",
+            questionId: 1,
+            responderName: "Erin Deeds",
+            sentiment: 0.5,
+    
+        },
+        {
+            answer: "I love opossums. I have rehabilitated baby opossums for 25 years, and I intend to do so until my last day!",
+            questionId: 1,
+            responderName: "Job Johnson",
+            sentiment: 0.8
+        },
+    
+        {
+            answer: "I always thought they were sort of nasty creatures...",
+            questionId: 2,
+            responderName: "Erin Deeds",
+            sentiment: 0.2
+    
+        },
+        {
+            answer: "Opossums are very misunderstood. People think they are dirty and diseased, but their drooling and hissing is a defense mechanism. They eat all kinds of pests, like ticks and mice, keeping disease down. They are wonderful critters!",
```

Comment:
Opossums are definitely misunderstood.

Reply:
Glad I could educate someone on the wonders of the North American opossum!